src/mjolnir/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ valhalla_module(NAME mjolnir
     valhalla::baldr
     SpatiaLite::SpatiaLite
     SQLite3::SQLite3
+    Boost::boost
```

Comment:
really doubt this is the missing piece to properly "link" the mjolnir exes but who knows..

Reply:
I recall I had to apply similar patch in the past, but then it wasn't necessary. I could not grasp what was going on, so I did not PR it.

Reply:
still not working on azure.. https://dev.azure.com/valhalla1/valhalla/_build/results?buildId=10260&view=logs&j=8c5a3053-cc94-5e24-da52-c43aabf65e0e&t=b45e0669-c54f-5dea-18c2-f681995b4f24

it seems like it's only affecting `mjolnir` executables though (i.e. `DATA_TOOLS`), `valhalla_validate_transit` just being the first trying to link libs. I have **no** idea what the problem could be: it's not CI caching and the only `mjolnir` specific instructions is about including luajit.. 

some win c++ lib requiring boost::regex in that exact msvc version? it **works locally** (or did a few commits ago when azure was failing exactly the same) and the only difference to CI setup could be a msvc version. I'll switch to win again sigh..

Reply:
tried again locally after upgrading to latest msvc (confirmed it's the same as on azure), `valhalla_validate_transit` still builds without issues.. out of ideas now.. if you any clue off the top of your head @mloskot on what to try/log, I'm all ears:)

Reply:
hm, one last thing to try is excluding all boost libs we don't need in the conan recipe (unfortunately they opted for `--without-xxx` logic rather than opt-in). it's possible that it's somehow messing smth up. and that's also the ultimate difference to my local setup: I never uninstalled boost from my local vcpkg. argh, that makes sense actually.. also there's a suspicious reference in the conan boost source where I'm not entirely sure what it's doing (even if it seems fairly unrelated): https://github.com/conan-io/conan-center-index/blob/306d3ab1bbf71aea6bf0f3f032fe03a420e376a0/recipes/boost/all/conanfile.py#L1062

failing that, I'm sorta tempted to just freaking install boost.regex with conan just to have CI pass and wait for people to come and complain.. terrible band-aid without knowing what's wrong..

Reply:
got the bastard, will comment more where it's more visible for future reference